README.md

```diff
@@ -6,4 +6,25 @@
 
 *Welcome to the Red Hat Observability Group Handbook*
 
-> In progress.
+## Goals
+
+For now check the [Handbook Proposal](content/Proposals/Accepted/202106-handbook.md)
+
+## Using
+
+This handbook was designed to be used in two ways:
+
+* Through our website `https://rhobs-handbook.netlify.app/`.
```

Comment:
It would be nice to use a Red Hat specific domain name instead of a third party one.

Reply:
IMO the effort required to acquire and serve the relevant certificate is almost certainly not worth the effort :disappointed:

Reply:
I would say the opposite. Branding this as Red Hat might need some special approvals and having this under "our" domain doesn't need anything like that.

Reply:
Yea, I would not bother for now, it might be not very important for us. (:

Reply:
The handbook is called Red Hat Observability Group Handbook, which to me already seems branded as Red Hat. 
I understand the limitations of obtaining a RH domain name, but I still netlify.app a bit unprofessional. Based on this proposal, the handbook will be accessible and used as a reference outside of the RHOBS group. 

As an alternative I would suggest using Github pages.

Reply:
Thanks, I see your point but:

* We can apply for a more branded website, there is a process we can start anytime. I don't see a blocker, but it will take time.
* What gives us more professionalism? Previously there was no documentation and only half baked Google Docs so I consider this a major improvement (: As the audience states we want primarily to have our team offloaded from knowledge repetitive in-meetings knowledge sharing by having our handbook. I don't think for our team it matters what domain it is. For external Red Hat teams too. For customers, maybe but they will rarely visit this page in current proposal.
* What's the difference between Netlify vs GitHub?

Reply:
> What's the difference between Netlify vs GitHub?

Slightly nicer URLs that are more like vanity URLs and a more globally recognized name that inspires some confidence. Otherwise not much really.

Reply:
If we are not going to use the domain outside of RH, I agree that it does not really matter much which one we use. 

> What's the difference between Netlify vs GitHub?

Netlify is yet another tool to learn, manage and request access for. With GH pages the user management model is tied to the repository. I also assume the Netlify account is a private one which makes things complicated when people leave the team. 

In any case, not a blocker in any way for the proposal, just wanted to raise some potential pitfalls.

Reply:
Yes, thanks! We have the netlify heavly used and relied on for years already on other project (Thanos, Observatorium, Prometheus,)(, that's why this choice. Once website will be more active we will ask for an open source tier which gives us team permissions. Added:

* https://github.com/rhobs/handbook/issues/14
* https://github.com/rhobs/handbook/issues/13